src/schema/ghga.yaml

```diff
@@ -219,6 +222,8 @@ classes:
           the sample was taken.
         required: true
         range: disease or phenotypic feature
+        exact_mappings: 
+          - EFO:0000651
```

Comment:
Same as above. Saying that a property is an exact match to a class would be incorrect. Instead, we can say `EFO:0000651 phenotype` is a narrow mapping to `disease or phenotypic feature`, via `narrow_mappings`.

In this case we are using `narrow_mappings` instead of `broad_mappings` because the term `EFO:0000651 phenotype` is more narrow in scope than `disease or phenotypic feature` (a grouping class for disease and phenotype).